dist/docsite/js/cdr-tokens.cjs

```diff
@@ -293,36 +293,106 @@ module.exports = {
+  "CdrProminenceElevatedShadow5Spread": "0",
+  "CdrProminenceFloatingShadow1Color": "rgba(75, 74, 72, 0.14)",
+  "CdrProminenceFloatingShadow1OffsetX": "1px",
+  "CdrProminenceFloatingShadow1OffsetY": "1px",
+  "CdrProminenceFloatingShadow1Blur": "4px",
+  "CdrProminenceFloatingShadow1Spread": "0",
+  "CdrProminenceFloatingShadow2Color": "rgba(75, 74, 72, 0.12)",
+  "CdrProminenceFloatingShadow2OffsetX": "5px",
+  "CdrProminenceFloatingShadow2OffsetY": "6px",
+  "CdrProminenceFloatingShadow2Blur": "8px",
+  "CdrProminenceFloatingShadow2Spread": "0",
+  "CdrProminenceFloatingShadow3Color": "rgba(75, 74, 72, 0.07)",
+  "CdrProminenceFloatingShadow3OffsetX": "11px",
+  "CdrProminenceFloatingShadow3OffsetY": "13px",
+  "CdrProminenceFloatingShadow3Blur": "10px",
+  "CdrProminenceFloatingShadow3Spread": "0",
+  "CdrProminenceFloatingShadow4Color": "rgba(75, 74, 72, 0.02)",
+  "CdrProminenceFloatingShadow4OffsetX": "19px",
+  "CdrProminenceFloatingShadow4OffsetY": "24px",
+  "CdrProminenceFloatingShadow4Blur": "12px",
+  "CdrProminenceFloatingShadow4Spread": "0",
+  "CdrProminenceFloatingShadow5Color": "rgba(75, 74, 72, 0)",
+  "CdrProminenceFloatingShadow5OffsetX": "30px",
+  "CdrProminenceFloatingShadow5OffsetY": "37px",
+  "CdrProminenceFloatingShadow5Blur": "13px",
+  "CdrProminenceFloatingShadow5Spread": "0",
+  "CdrProminenceLiftedShadow1Color": "rgba(75, 74, 72, 0.14)",
+  "CdrProminenceLiftedShadow1OffsetX": "5px",
+  "CdrProminenceLiftedShadow1OffsetY": "2px",
+  "CdrProminenceLiftedShadow1Blur": "11px",
+  "CdrProminenceLiftedShadow1Spread": "0",
+  "CdrProminenceLiftedShadow2Color": "rgba(75, 74, 72, 0.12)",
+  "CdrProminenceLiftedShadow2OffsetX": "19px",
+  "CdrProminenceLiftedShadow2OffsetY": "7px",
+  "CdrProminenceLiftedShadow2Blur": "20px",
+  "CdrProminenceLiftedShadow2Spread": "0",
+  "CdrProminenceLiftedShadow3Color": "rgba(75, 74, 72, 0.07)",
+  "CdrProminenceLiftedShadow3OffsetX": "43px",
+  "CdrProminenceLiftedShadow3OffsetY": "16px",
+  "CdrProminenceLiftedShadow3Blur": "28px",
+  "CdrProminenceLiftedShadow3Spread": "0",
+  "CdrProminenceLiftedShadow4Color": "rgba(75, 74, 72, 0.02)",
+  "CdrProminenceLiftedShadow4OffsetX": "77px",
+  "CdrProminenceLiftedShadow4OffsetY": "28px",
+  "CdrProminenceLiftedShadow4Blur": "33px",
+  "CdrProminenceLiftedShadow4Spread": "0",
+  "CdrProminenceLiftedShadow5Color": "rgba(75, 74, 72, 0)",
+  "CdrProminenceLiftedShadow5OffsetX": "120px",
+  "CdrProminenceLiftedShadow5OffsetY": "44px",
+  "CdrProminenceLiftedShadow5Blur": "36px",
+  "CdrProminenceLiftedShadow5Spread": "0",
+  "CdrProminenceFlat": [{"color":"#000000","offsetX":"0","offsetY":"0","blur":"0","spread":"0"}],
+  "CdrProminenceRaised": [{"color":"rgba(75, 74, 72, 0.07)","offsetX":"0","offsetY":"1px","blur":"1px","spread":"0"},{"color":"rgba(75, 74, 72, 0.02)","offsetX":"0","offsetY":"2px","blur":"1px","spread":"0"},{"color":"rgba(75, 74, 72, 0.00)","offsetX":"0","offsetY":"2px","blur":"1px","spread":"0"}],
```

Comment:
I would expect this to be a single line, rather than an array—what do you folks think? These variables are definitely less used, but I was under the impression these should be a single line and could be used as-is by a consumer without needing to join an array.

Reply:
Like a CSS value?

Reply:
Ya I think so

Reply:
my understanding was that these would be used to create a prominence composite is that not the case?